pkg/prometheus/promcfg.go

```diff
@@ -2257,29 +2261,34 @@ func (cg *ConfigGenerator) appendEvaluationInterval(slice yaml.MapSlice, evaluat
 	return append(slice, yaml.MapItem{Key: "evaluation_interval", Value: evaluationInterval})
 }
 
+func (cg *ConfigGenerator) appendGlobalLimits(slice yaml.MapSlice, limitKey string, limit *uint64, enforcedLimit *uint64) yaml.MapSlice {
+	if ptr.Deref(limit, 0) > 0 {
```

Comment:
should we check that limit <= enforced? Otheriwse is there a risk of the "soft" limit being higher than the enforced limit?

Reply:
@simonpasquier wdyt?